### DISCUSSION TITLE
关于打开【声音增强功能和空间音频】该软件只显示一个主声道

### DISCUSSION THREAD

**Discussion by Git-OuO** [File-New-Project/EarTrumpet#1283] [Q&A] [answered]
打开【Windows声音增强功能和空间音频】该软件只显示一个主声道。
turning on [Windows Sound Enhancement and Spatial Audio] The software displays only one master channel.

**Comment by riverar** [comment, marked as answer]
This is a known issue https://github.com/File-New-Project/EarTrumpet/issues/1305, thanks!
